php/ext/google/protobuf/encode_decode.c

```diff
@@ -887,6 +978,78 @@ static void* oneofsubmsg_handler(void* closure, const void* hd) {
   return submsg;
 }
 
+// Sets a non-repeated wrapper submessage field in a message.
+static void* wrapper_submsg_handler(void* closure, const void* hd) {
+  MessageHeader* msg = closure;
+  const submsg_handlerdata_t* submsgdata = hd;
+  TSRMLS_FETCH();
```

Comment:
This may not be needed if those TSRM macros are not needed for the rest of the function? Or maybe it's needed after some other macros got expanded? please confirm.

Reply:
UNBOX_HASHTABLE_VALUE needs it